blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/transport/OrbManagement.java

```diff
@@ -26,9 +26,9 @@
 public class OrbManagement implements Runnable {
     private static final Logger log = LogManager.getLogger(OrbManagement.class);
     private static final String CorbaOrbClassProp = "org.omg.CORBA.ORBClass";
-    private static final String CorbaOrbClassValue = "org.jacorb.orb.ORB";
+    private static final String CorbaOrbClassValue = "com.sun.corba.se.impl.orb.ORBImpl";
```

Comment:
Thanks a lot for your input !

I'd reviewed the codebase and found that https://github.com/jbosstm/narayana/blob/master/blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/tx/TransactionImpl.java which also uses the OrbManagement. I'm not very sure that we should remove it ?